src/lang/db.pl

```diff
@@ -38,15 +38,15 @@
 :- setting(drop_graphs, list, [user],
 		'List of named graphs that should initially by erased.').
 
-%%
-:- multifile collection_name/1.
-
 %%
 annotation_property('http://www.w3.org/2000/01/rdf-schema#comment').
 annotation_property('http://www.w3.org/2000/01/rdf-schema#seeAlso').
 annotation_property('http://www.w3.org/2000/01/rdf-schema#label').
 annotation_property('http://www.w3.org/2002/07/owl#versionInfo').
 
+collection_name(Name) :-
+	mng_db_name(DB), mng_collections(DB, Collections), !, member(Name, Collections).
```

Comment:
This fix is problematic, because you don't want to always import or export all collections you have. The ones that are necessary are most of the time only triples and tf. 

At the moment the user could choose the collections by asserting collection_name themselves. I think i could either add a setting in which the user can have a list of the collections he wants to ex- or import (default would be triple and tf), or one could just fix this two collections here and allow the user to assert additional collections.

Reply:
Sounds good. I removed it from this PR, but I think that this should still be somehow addressed. Currently, the default behavior of remember/1 is to not import anything, because collection_name is always false. So adding default values/a setting for collection_name would be very useful.